hack/golangci-lint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+
+paths=""
+while IFS= read -r line; do
+    # read directory from the file and append a wildcard
+    paths+="${line}/... "
+done <hack/lint-paths.txt
+
+golangci-lint run --timeout 20m --verbose ${paths} &&
+    golangci-lint run --disable-all -E ginkgolinter --timeout 10m --verbose --no-config \
+        ./pkg/... \
+        ./tests/...
```

Comment:
Since the script fails on error (`set -e`), there is no need condition the execution of the second command using the `&&` operator:

```suggestion
golangci-lint run --timeout 20m --verbose ${paths}

golangci-lint run --disable-all -E ginkgolinter --timeout 10m --verbose --no-config \
./pkg/... \
./tests/...
```

Reply:
done